content/en/docs/concepts/storage/volumes.md

```diff
@@ -835,7 +835,9 @@ You must have your own Quobyte setup running with the volumes
 created before you can use it.
 {{< /caution >}}
 
-See the [Quobyte example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/quobyte) for more details.
+Quobyte supports [Container Storage Interface](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/).
```

Comment:
```suggestion
Quobyte supports the {{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}}.
```
I'm not sure whether the website style guide prefers CSI to Container Storage Interface there, but it doesn't read as well anyway. The [glossary entry](https://kubernetes.io/docs/reference/glossary/?all=true#term-csi) is a good place to link to and will have up-to-date signposting to further reading etc.